src/calculateRentalCost.js

```diff
@@ -29,6 +29,22 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let totalAmount = 0;
+  const rentPerDay = 40;
+  const sevenDaysDiscount = 50;
+  const threeDaysDiscount = 20;
+
+  for (let i = 0; i < days; i++) {
+    totalAmount += rentPerDay;
+  }
+
+  if (days >= 3 && days < 7) {
+    totalAmount -= threeDaysDiscount;
+  } else if (days >= 7) {
+    totalAmount -= sevenDaysDiscount;
+  }
```

Comment:
You should return the value immediately in if statements.

```suggestion
  if (days >= 3 && days < 7) {
    totalAmount -= threeDaysDiscount;
    
    return totalAmount;
  } else if (days >= 7) {
    totalAmount -= sevenDaysDiscount;
    
    return totalAmount;
  }
```